designs/administrator/4.8/getting-started-card/index.md

```diff
@@ -0,0 +1,67 @@
+---
+parent: Administrator
+version: 4.8
+---
+
+# Getting started card
+
+## Background
+A quick starts card exists today on the following landing pages for the Admin and Developer perspectives with the goal to help users learn and use resources in the Console.
+
+- Admin Cluster Overview dashboard
+
+![cluster-overview](img/quick-starts-cluster-overview.png)
+
+- Developer Add page 
+
+![add-page](img/quick-starts-add.png)
+
+## Goal
+As our getting started efforts expand, we realize that we can capture more than just quick starts to help users get started. The goal of this larger Getting Started card is to supply users with many different pathways to learn and use resources in the Console. This card will replace the existing quick start card on both the Add page and Cluster Overview.
+
+## Card design and interactions
+The design of the card will remain consistent on the Admin and Developer perspectives, but the content within each column will vary. The reason for this is because there is featured content we would like users to see depending on the perspective they land on.
+
+![card](img/card-design.png)
+
+A popover next to the card title will explain more about what "Getting Started" means in this context. The card should also be able to be hidden, considering there are going to be users who do not want to see it. 
+
+![card-interactions](img/card-interactions.png)
+
+### Content guidance
+
+There should be no more than two items per category, in an attempt to not completely overwhelm the user. Each item is (and should be) actionable, no matter the users' permissions. The link text at the bottom should be sending the user to a more general space for that category. For example, "Build with guided documentation" lists out quick starts. We added a "View all quick starts" link at the bottom to send the user to our quick start catalog page so that they can see all that exist.
+
+### Hiding and adding the card
+
+A kebab in the top right corner of the card will allow the user to hide the card.
+
+![add-page-dismiss](img/add-page-dismiss.png)
+![cluster-overview-hidden](img/cluster-overview-hidden.png)
+
+Once the card is hidden, a label will appear in the page header, allowing the user to bring it back. This label acts more as a "hint" and is dismissible itself, in case the user has zero interest in seeing anything getting started related.
```

Comment:
I wonder if some sort of like link or expandable control could be used in place of a label here? The use here sort of feels like more prominent than I'd expect for something the user just expressed interest in hiding. Perhaps if it was more discrete we would also not need a way to remove that control entirely, maybe addressing always giving a way to add this card back.

Also selfishly I know we are using a similar label to highlight quick starts on pages they relate to so I don't want the user to start to ignore them. 😁

Reply:
+1 to Peter's comment. At least on the developer side, I think we need to come up with a more holistic solution for showing and hiding. I'd like to see the "show getting started label" and the "Details on" toggle combined.

On the Topology view, we have a display dropdown that lets the user configure their display. It has different controls within the dropdown. Might be a good area to get some user feedback post 4.8 on different options...

Reply:
@itsptk It's too late to change that for this first pass since it's already been approved by stakeholders BUT this issue should be solved in future iterations by one of the following:
- User can access Getting started resources in the help menu , where that catalog page would replace the existing Quick starts page. That way you can access the information on the page rather than bringing the card back
- Use some sort of toggle (like @beaumorley was saying) in page heading to allow for the card to be brought back (we would combine this with the details on/off switch on the developer side)
- When customizable dashboards are introduced, which both Admin and Dev PMs have already been discussing, we can include the ability to hide/show the card and move it around with that functionality

Hopefully that helped! I can add those ideas in the future section as well